### PR TITLE
Update regression tests

### PR DESCRIPTION
**Pull request by yifanz0**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/RegressionTests/adls_setaclresusive.ps1 [hunk 1]
```diff
@@ -44,13 +44,13 @@ BeforeAll {
...
     $resourceGroupName = $globalNode.resourceGroupName
-    $storageAccountKey = (Get-AzStorageAccountKey -ResourceGroupName $resourceGroupName -Name $testNode.accountName)[0].Value
+    $storageAccountKey = $testNode.accountKey
 
     $ctx = New-AzStorageContext  $testNode.accountName -StorageAccountKey $storageAccountKey
     $ctx2 = New-AzStorageContext  $testNode.accountName
 
     $filesystemName = "adlstest2"
-    $localSrcFile = "C:\temp\testfile_1K_0" #The file needs to exist before tests, and should be 512 bytes aligned
+    $localSrcFile = ".\data\testfile_1024K_0" #The file needs to exist before tests, and should be 512 bytes aligned
```
Comment (blueww) [inline src/Storage/RegressionTests/adls_setaclresusive.ps1:53]: Why change it to 1024K?

**Review by blueww — APPROVED** [review]

File: src/Storage/RegressionTests/adls_setaclresusive.ps1 (hunk 1 above)
Comment (yifanz0) [reply src/Storage/RegressionTests/adls_setaclresusive.ps1:53]: Sorry had a typo here when changing the file path. Changed it back

**Comment by blueww** [comment]
BTW, for the data need prepare: 
Following looks a little duplicated (especially for the 2nd, we might don't want to keep 2 big files):

1. testfile_10M , testfile_10240K_0
2. testfile_300M, testfile_300000K_0

> 	| testfile_1K_0      | 1KB     |
> 	| testfile_10M       | 10M     |
> 	| testfile_300M      | 300M    |
> 	| testfile_2048K     | 2048K   |
> 	| testfile_10240K_0  | 10240K  |
> 	| testfile_300000K_0 | 300000K |

**Comment by yifanz0** [comment]
@blueww  Yes indeed. I'll address the duplicates in the next PR. 300M test file was added only for the blob copy cases to test block to block copy when size is bigger than 256M

**Comment by blueww** [comment]
> @blueww Yes indeed. I'll address the duplicates in the next PR. 300M test file was added only for the blob copy cases to test block to block copy when size is bigger than 256M

Thanks for the reply!
I think we just need to keep 1 test file > 256MB.

**Pull request merged**